### ISSUE TITLE
SLOWfast_RCNN migration learning

### ISSUE THREAD

**Issue by ysy070100** [open-mmlab/mmaction2#1758]
I can run the training normally, but the environment used by the model is mainly the monitoring environment, but the amount of data is not large, so I want to save the five types of action recognition functions corresponding to the training model (11:sit, 12:stand, 14:walk, 29:drink, 27:eat), and realize the migration learning. Please ask me what I should do?

**Comment by hukkai** [comment]
@ysy070100 You can refer to ``Train Custom Classes From Ava Dataset`` the section at https://github.com/open-mmlab/mmaction2/blob/master/configs/detection/ava/README.md.

**Comment by ysy070100** [comment]
@hukkai In the link you sent, if I don't understand the process of less category training correctly, it should be based on the original data set for less category training. Download the original data set and only select some of the categories for training. Finally, get the recognition model of the corresponding action, and improve the accuracy of action recognition.

I want to have the recognition effect of the five types of actions selected from the 80 categories, and add the action recognition effect of these five types of actions on the monitoring perspective. Now I think about two schemes:

1. Download the AVA dataset, and add my own labeled files and corresponding RGB frames in the rawframes and annotation files. These my own annotations (1:sit, 2:stand, 3:walk, 4:drink, 5:eat) should be modified into the corresponding annotation number format in the AVA dataset (11:sit, 12:stand, 14:walk, 29:drink, 27:eat),

2. Only use my own data, do not download the AVA data set, and transfer the recognition effect of these five types of actions (11:sit, 12:stand, 14:walk, 29:drink, 27:eat) in the pre training model (I may not express it accurately in this place)

If it is scheme 2, how should I complete it?

**Comment by ysy070100** [comment]
@hukkai Because in my self-made dataset, action_ list. The annotation in pbtxt file is like this (1:sit, 2:stand, 3:walk, 4:drink, 5:eat), which is different from the digital annotation in the pre training model (11:sit, 12:stand, 14:walk, 29:drink, 27:eat). Does this mean that the action recognition effect in the pre training model will not be loaded into my own training model? If so, what effect will this pre training model have

**Comment by hukkai** [comment]
You can download the pre-train checkpoint trained on AVA from our repo, and fine-tune the model with your data. You can also freeze the network backbone and train the classifier only.

**Comment by ysy070100** [comment]
> re-train checkpoint trained on AVA from our repo, and fine-tune the model with your data. You can also freeze the network backbone and train the classifier only.

@hukkai  How can I  fine-tune the model with my data?Maybe this method can achieve the effect I want. Could you please give me a slightly more detailed answer or relevant documents？Thanks.

**Comment by ysy070100** [comment]
> You can also freeze the network backbone and train the classifier only.

@hukkai Do you mean transfer learning?

Is there a guide document for migration learning in mmaction2?

**Comment by hukkai** [comment]
@ysy070100 You can look at the `Modify Training Schedule` section here `tutorial 2: finetuning models`: https://mmaction2.readthedocs.io/en/latest/tutorials/2_finetune.html#modify-training-schedule

**Comment by hukkai** [comment]
If you need any further discussions, feel free to reopen the issue. Thanks!
